04.ls/ls.rb

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+require 'optparse'
 require_relative 'lib/ls_command'
 
+option = ARGV.getopts('a').transform_keys(&:to_sym)
 path = ARGV[0]
 
-LsCommand.new(path).execute
+LsCommand.new(option, path).execute
```

Comment:
```suggestion
LsCommand.new(path, option).execute
```

optionは最後の方に来ることが多いですね。文字通りオプションなので。